xarray/plot/facetgrid.py

```diff
@@ -282,13 +283,31 @@ def __init__(
         self._mappables = []
         self._finalized = False
 
+    @property
+    def axes(self) -> np.ndarray:
+        warnings.warn(
+            "self.axes is deprecated, use self.axs instead.",
```

Comment:
```suggestion
            "self.axes is deprecated in order to align with matplotlibs plt.subplots, use self.axs instead.",
```
Was it something like this you had in mind @mathause ? I'll add in a what's new as well eventually.

Reply:
done.